openmc/plots.py

```diff
@@ -250,7 +250,7 @@ def voxel_to_vtk(voxel_file: PathLike, output: PathLike = 'plot.vti'):
         writer.SetInputData(grid)
     else:
         writer.SetInput(grid)
-    if not output.endswith(".vti"):
+    if not output.name.endswith(".vti"):
```

Comment:
This is subtle, but the `PathLike` used here isn't the same as `os.PathLike`. A different `PathLike` is imported at the top of this file that is the union of `str` and `os.PathLike`. This means that either a string or `os.PathLike` should be valid to pass as an argument for `output`. In the case that `output` is passed as a string, this change will cause an error due to the missing object attribute.

Given that we convert `output` to a string below when passing it to `vtkXMLImageDataWriter.SetFileName`, I'd recommend that we simply do so earlier in the function to fix this issue and revert this change.

Reply:
Sounds good -- I'll push an update that does the string conversion based on type earlier. I'll take a look at how it is being done elsewhere and propose something in the PR.

Reply:
Just a simple `output = str(output)` above this line should work (and then change it back to `if not output.endswith`)

Reply:
Done in d532e0d